app/routes/_index.tsx

```
@@ -1,7 +1,8 @@
 import { json, type LoaderFunction, type MetaFunction } from "@remix-run/node";
-import { CurrentMusicComponent } from "~/components/CurrentMusic/";
+import { CurrentlyPlayingCard } from "~/components/CurrentMusic/CurrentMusic";
 import { LastPlayedSongs } from "~/components/LastPlayedSongs";
```

ESLint

[error] 'LastPlayedSongs' is defined but never used.

(@typescript-eslint/no-unused-vars)